### ISSUE TITLE
Accessibility: flying bird gif should have a text alternative

### ISSUE THREAD

**Issue by ameliejyc** [FAC-11/week4-3to5#63]
One accessibility audit warning:

[Warning] Images should have a text alternative or presentational role (1)
See AX_TEXT_02 for more information.
<img class=​"ho-oh" src=​"http:​/​/​pokestadium.com/​sprites/​xy/​ho-oh.gif">​
